bot/code_review_bot/workflow.py

```diff
@@ -191,6 +197,41 @@ def publish(self, revision, issues, task_failures, links):
         )
         stats.add_metric("analysis.issues.publishable", nb_publishable)
 
+        if self.lando is not None:
+            logger.info(
+                "Publishing warnings to lando for {0} errors and {1} warnings".format(
+                    nb_publishable_errors, nb_publishable - nb_publishable_errors
+                ),
+                revision=revision.id,
+                diff=revision.diff["id"],
+            )
+            self.lando.del_all_warnings(revision.id, revision.diff["id"])
+
+            # Determine if we need to post something or not
+            if nb_publishable > 0:
+                lando_warning_message = (
+                    "There are still"
+                    + (
+                        " {} errors".format(nb_publishable_errors)
+                        if nb_publishable_errors
+                        else ""
+                    )
+                    + (
+                        " and"
+                        if (nb_publishable_errors and nb_publishable_warnings)
+                        else ""
+                    )
+                    + (
+                        " {} warnings.".format(nb_publishable_warnings)
+                        if nb_publishable_warnings
+                        else ""
+                    )
+                    + " Are you sure you want to continue?"
+                )
+                self.lando.add_warning(
+                    lando_warning_message, revision.id, revision.diff["id"]
+                )
```

Comment:
Instead of having this code here, it'd be cleaner to build a new `Reporter` class (like we have the `PhabricatorReporter`), e.g. `LandoReporter`.

Reply:
I thought about that, but we would have a special case when no matter if we have or don't have issues pending we will need to do reporter.publish()

Reply:
Yes, but that's not a problem, the PhabricatorReporter also does that check